internal/controllers/clustertemplate_controller.go

```diff
@@ -339,3 +416,24 @@ func (r *ClusterTemplateReconciler) enqueueClusterTemplatesForConfigmap(ctx cont
 	}
 	return requests
 }
+
+// mapKeysToSlice takes a map[string]bool and returns a slice of strings containing the keys
+func mapKeysToSlice(inputMap map[string]bool) []string {
+	keys := make([]string, 0, len(inputMap))
+	for key := range inputMap {
+		keys = append(keys, key)
+	}
+	sort.Strings(keys)
+	return keys
+}
+
+// sliceToString takes a slice of strings an returns a comma separated string including the slice content
+func sliceToString(aSlice []string) (out string) {
```

Comment:
You should be able to use `strings.Join` instead of this function.

Reply:
Fixed